projects/origin-release/rhel-7-golang-1.12/rhel-server-rhscl.repo

```diff
@@ -0,0 +1,10 @@
+[rhel-server-rhscl]
+name = rhel-server-rhscl
+baseurl = https://use-mirror1.ops.rhcloud.com/enterprise/reposync/4.1/rhel-server-rhscl-rpms/
+          https://use-mirror2.ops.rhcloud.com/enterprise/reposync/4.1/rhel-server-rhscl-rpms/
+sslverify = false
+sslclientkey = /tmp/key/ops-mirror.pem
```

Comment:
@stevekuznetsov Save to assume these are mounted or no?

Reply:
Yes. What image is using this config to serve RPMs?